### PR TITLE
Add two simple services to process topological-inventory payloads from the upload service

### PR DESCRIPTION
**Pull request by agrare**
This adds a simple validation service and a processing service to handle topological-inventory and/or cfme payloads sent to the upload service.

### CODE REVIEW

File: bin/topological-inventory-inventory-upload [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+STDOUT.sync = true
+
+require "bundler/setup"
+require "topological_inventory/sync/inventory_upload_worker"
+
+def parse_args
+  require 'optimist'
+  opts = Optimist.options do
+    opt :queue_host, "Hostname of the Platform's kafka queue", :type => :string,
+        :default => ENV["QUEUE_HOST"] || "localhost"
```
Comment (Fryguy) [inline bin/topological-inventory-inventory-upload]: Minor, but I think it's preferable to do `ENV.fetch("QUEUE_HOST", "localhost")`

Comment (agrare) [reply bin/topological-inventory-inventory-upload]: Okay done, we do this literally everywhere so we have lots to change then

**Comment by carbonin** [comment]
@agrare do you need/want to create a separate bin script for this like the sources sync one? Either way I don't think we'll need a separate dockerfile. We should be able to set command or args to whatever we need to to make the same image run the other service.

Also, I assume neither of these will need the database?

**Comment by agrare** [comment]
> do you need/want to create a separate bin script for this like the sources sync one

There should already be a bin script for each one of these services

> Also, I assume neither of these will need the database?

Correct, neither of these need db access.  They will both need kafka host/port and `topological-inventory-inventory-upload ` will need ingress_api info.

**Comment by carbonin** [comment]
> There should already be a bin script for each one of these services

Ah sorry, that was confusing. I was talking about https://github.com/ManageIQ/topological_inventory-sync/blob/master/docker-assets/run_sources_sync, not https://github.com/ManageIQ/topological_inventory-sync/blob/master/bin/topological-inventory-sources-sync when I said "bin-script", but since they don't need the database and don't need to decrypt passwords I don't think we'll need one.

**Comment by agrare** [comment]
@carbonin right we should be able to just set the ENTRYPOINT dynamically like we do with openshift-collector/openshift-operations ?

**Comment by carbonin** [comment]
Yup, looks like that would be the way to go @agrare

**Comment by agrare** [comment]
Thanks @carbonin 

@Fryguy can you take another look?

**Pull request merged**